lightly_studio/src/lightly_studio/core/image/image_sample.py

```
@@ -42,6 +42,13 @@ def __init__(self, inner: ImageTable) -> None:
         self.inner = inner
         super().__init__(sample_table=inner.sample)
 
+    def __str__(self) -> str:
```

Ruff (0.15.12)

[warning] Missing docstring in magic method

(D105)